src/docsfy/api/projects.py

```
@@ -1169,6 +1203,18 @@ def _sort_key(v: dict[str, Any]) -> str:
 # ---------------------------------------------------------------------------
 
 
+async def _load_available_models() -> dict[str, list[dict[str, str]]]:
+    """Load available models for all providers from ai-cli-runner's model cache."""
+    result: dict[str, list[dict[str, str]]] = {}
+    for provider in VALID_PROVIDERS:
+        try:
+            result[provider] = await model_cache.list_models(provider)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)